docs/blog/2019-02-26-getting-started-with-gatsby-themes/index.md

```diff
@@ -250,7 +250,7 @@ export default ({ children }) => (
 
 To import your styled components, go to _index.js_
 
-`javascript:title=packages/theme/index.js`
+`packages/theme/index.js`
```

Comment:
This seems to be a little more in spirit of what was intended. Maybe try this?

```suggestion
```js:title=packages/theme/index.js
packages/theme/index.js
```
```